components/transaction/internal/adapters/database/postgres/assetrate.postgresql.go

```diff
@@ -0,0 +1,73 @@
+package postgres
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/LerianStudio/midaz/common"
+	"github.com/LerianStudio/midaz/common/mpostgres"
+	a "github.com/LerianStudio/midaz/components/transaction/internal/domain/assetrate"
```

Comment:
please, use **ar** for **asset_rate** because we are using **a** for **accounts**, just to normaliza the imports.